templates/bastion.yaml.tmpl

```diff
@@ -30,6 +30,79 @@ systemd:
         ExecStart=/usr/sbin/sysctl -p /etc/sysctl.d/hardening.conf
         [Install]
         WantedBy=multi-user.target
+    - name: docker.service
+      enabled: true
+      dropins:
+      - name: 10-giantswarm-extra-args.conf
+        contents: |
+          [Service]
+          Environment=DOCKER_OPTS="--log-opt max-size=50m --log-opt max-file=2"
+          Environment=DOCKER_OPT_BIP=--bip={{ .DockerCIDR }}
+    - name: node-exporter.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=node-exporter
+        Requires=docker.service
+        After=docker.service
+        [Service]
+        Restart=on-failure
+        RestartSec=1
+        StartLimitInterval=300s
+        StartLimitBurst=3
+        TimeoutStartSec=0
+        Environment="IMAGE=quay.io/giantswarm/node-exporter:v0.18.0"
```

Comment:
can we go straight to 1.0.1 ?

Reply:
I've been testing 1.0.1 - i must've overwritten my changes whilst sorting some syntax issues i introduced. I've updated it to 1.0.1 :+1: